programs/ssl/ssl_client2.c

```diff
@@ -1225,7 +1320,11 @@ int main( int argc, char *argv[] )
         mbedtls_printf( USAGE1 );
         mbedtls_printf( USAGE2 );
         mbedtls_printf( USAGE3 );
+#if defined(MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL) 
```

Comment:
It's easier to read if the relevant configuration dependent logic is within the definition of `USAGE_X`, so we can simply print all `USAGE_X` strings here.